### PR TITLE
stim_sync & talent_checker review,  minionsize_sync creation

### PR DESCRIPTION
**Pull request by ercete**
TL;DR; : 
*  Minion quantity status icon refers now to 'actual' minion quantity instead of 'maximum'.
*  Fixed double menu entry for 'Stimpack syncing status icon', the config menu as migrated in 'Talent Checker' submenu.
*  New status icons for Stimpacks, Minions and Minions at zero.

Details : 
* Replace old minionsize by minion_sync matching actual minion quantity.
* Minion_sync & stim sync configurations are now set in talent_checker submenu to avoid double config lines.
* 'source' has been renamed to hookevent names.
* update_status has been reviewed to work with multiple counters
* New icons for stim_sync, minionsize and minionskull & update artwork README.md
* Sorry for single-commit

### CODE REVIEW

**Review by wrycu — COMMENTED** [review]
Can you give a TL;DR of the changes you've made so I can include them in the readme?

**Comment by wrycu** [comment]
FYI @bheiskell since this is a big change and you're kinda a professional coder

**Comment by ercete** [comment]
What about :
* Minion quantity status icon refers now to 'actual' minion quantity instead of 'maximum'.
* Fixed double menu entry for 'Stimpack syncing status icon', the config menu as migrated in 'Talent Checker' submenu.
* New status icons for Stimpacks, Minions and Minions at zero.

**Comment by ercete** [comment]
Ow !
I forgot to test if the 'adversary' status icon (handled by talent checker) is still working well.
I'll check it soon.

**Comment by wrycu** [comment]
> What about :
> 
> * Minion quantity status icon refers now to 'actual' minion quantity instead of 'maximum'.
> * Fixed double menu entry for 'Stimpack syncing status icon', the config menu as migrated in 'Talent Checker' submenu.
> * New status icons for Stimpacks, Minions and Minions at zero.

Sounds good. FYI, you can hit the three dots next to the comment to quote it for your reply so the conversation is a tad easier to follow.

**Comment by ercete** [comment]
I checked : "Adversary" still looks to be OK.

I don't understand why there is a conflict on fr.json now...
